python/ngraph/utils/types.py

```diff
@@ -32,6 +32,7 @@
 TensorShape = List[int]
 NumericData = Union[int, float, np.ndarray]
 NumericType = Union[type, np.dtype]
+ScalarType = Union[int, float]
```

Comment:
This should be named `ScalarData`, by analogy to `NumericData` above.

Reply:
Fixed.